tests/e2e/tidbcluster/stability.go

```diff
@@ -473,7 +473,7 @@ var _ = ginkgo.Describe("[Stability]", func() {
 					if apierrors.IsNotFound(err) {
 						continue
 					}
-					if pvc.Spec.StorageClassName != nil && *pvc.Spec.StorageClassName == "local-storage" {
+					if pvc.Spec.StorageClassName != nil && *pvc.Spec.StorageClassName == "standard" {
```

Comment:
This is for AWS, should we leave this untouched?

Reply:
fixed.